qemu/tests/cfg/smbios_default_check.cfg

```diff
@@ -0,0 +1,24 @@
+- smbios_default_check: install setup image_copy unattended_install.cdrom
+    only q35
+    no Host_RHEL.m6,Host_RHEL.m7,Host_RHEL.m8.u0,Host_RHEL.m8.u1
+    # dmidecode not available (and no plans for it) on s390
+    no s390x
+    type = smbios_default_check
+    kill_vm = yes
+    required_qemu= [4.2.0,)
+    # Expected output
+    System_Manufacturer = "^Red Hat$"
+    System_SKU_Number = "^8.[23456789].0|9.\d.0$"
+    Baseboard_Manufacturer = "^Red Hat$"
+    Baseboard_Product_Name = "^RHEL-AV$"
+    # Command to get BIOS info
+    RHEL:
```

Comment:
Should Line 10-13 belong to `RHEL:` too?

Reply:
> Should Line 10-13 belong to `RHEL:` too?

Oh no, should be `Host_RHEL`.

Reply:
No, lines 10-13 are the expected results, it will not change due to platform changes.Thanks1

Reply:
@leidwang So you tried this case on Fedora host?

Reply:
No,it test on rhel host.

Reply:
@leidwang Either you restrict the users run this test on the other host or you support them. So I recommend you only define those parameters for `Host_RHEL`, to force fedora or other OS users define their ones.

Reply:
Ok,I understand your thought, I also agree with this,so I will add "`HOST_RHEL`" to limit those parameters only work on RHEL host.Thanks!